menas/pom.xml

```diff
@@ -240,10 +238,7 @@
                             <filtering>true</filtering>
                         </resource>
                     </webResources>
-                    <packagingExcludes>
-                        META-INF/*.SF,
-                        META-INF/*.DSA,
-                        META-INF/*.RSA
+                    <packagingExcludes> META-INF/*.SF, META-INF/*.DSA, META-INF/*.RSA 
```

Comment:
If we are inlining this, let's bring up the closing tag to be on the same line as well.